.hound_config.ini

```diff
@@ -0,0 +1,3 @@
+[flake8]
+max-line-length = 160
+ignore = E111
```

Comment:
I take back my advice to simply ignore this. Sorry! :smile: I thought this was configurable in flake8, but it's not: https://groups.google.com/forum/#!topic/comp.lang.python/BkiEex5RsFs. 

The most important thing is consistency: you shouldn't have some files with two-space indentation and others with four. If you can't configure this tool to support two spaces, better to standardize on four. (It's pretty unusual to see two-space indentation in the Python world anyways).